src/adaptix/_internal/morphing/facade/retort.py

```diff
@@ -154,6 +159,10 @@ class FilledRetort(OperatingRetort, ABC):
 
         ABCProxy(Mapping, dict),
         ABCProxy(MutableMapping, dict),
+
+        ConcatProvider()
+        if NO_BYTE_STRING
+        else
         ABCProxy(ByteString, bytes),
```

Comment:
It is looks very odd. It is similar to four separte list items.

```suggestion
         (
             ABCProxy(ByteString, bytes) if HAS_BYTE_STRING else ConcatProvider()
         ),
```

or

```suggestion
         ABCProxy(ByteString, bytes) if HAS_BYTE_STRING else ConcatProvider(),
```